hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -287,34 +287,17 @@
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
                   rel_path: "hw/ip/tlul/sram2tlul/lint/{tool}"
              },
-             {    name: top_earlgrey
-                  fusesoc_core: lowrisc:systems:top_earlgrey
-                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
-                  rel_path: "hw/top_earlgrey/lint/{tool}"
-                  overrides: [
-                    {
-                      name: design_level
-                      value: "top"
-                    }
-                  ]
-             },
              {    name: chip_earlgrey_asic
                   fusesoc_core: lowrisc:systems:chip_earlgrey_asic
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
-                  rel_path: "hw/chip_earlgrey_asic/lint/{tool}"
+                  rel_path: "hw/top_earlgrey/lint/{tool}"
```

Comment:
@sriyerg I've removed the `top_earlgrey` run and renamed the path for `chip_earlgrey_asic` so that all chip-level tool runs share the same path stem.